Dockerfiles/midstream/run_tests.py

```diff
@@ -42,24 +42,45 @@ def run_validate_operator_bundle(self):
                     -e 'work_dir={work_dir}'".format(operator_dir=self.operator_dir,
                                                     operator_work_dir=self.operator_work_dir,
                                                     work_dir=self.work_dir)
-        result = subprocess.run(exec_cmd, shell=True)
+        result = subprocess.run(exec_cmd, shell=True, capture_output=True)
         return result
 
     def test_for_extract_and_validate_bundle_image(self):
+        
+        self.setUp()
+        global exit_code
+        exit_code = 0
         result = self.run_extract_operator_bundle()
-        self.assertEqual(0,
-                         result.returncode,
-                         "extract-operator-bundle.yml failed")
+        if(result.returncode != 0):
+            print("Ansible playbook extract-operator-bundle.yml failed with result code: %s , see the file extract-operator-bundle.errormessage for more info." % result.returncode)
+            with open("extract-operator-bundle.errormessage", "w") as errorMsg:
+                print("Result code: " + str(result.returncode), "Error message: " + str(result.stderr), file=errorMsg)
+            exit_code = 102
```

Comment:
IIUC we will always return exit code 102 for any failure detected by the playbook extract-operator-bundle.yml.
But requirements says we should return 50-99.

We need to return exit code 102 just for invalid "input" env variable

Reply:
Currently I don't know of a different way to simulate test : test_negative_validate_operator_bundle()
Other then setting wrong work dir.

Reply:
We need to work on image that pass extraction but fails on validation, we currently don't correspond with such image.

Reply:
@jsztuka let me know if you need help creating an image that satisfies these requirements. The quay.io/cvpops/test-operator:invalid-dependencies-v1 image passes extraction and fails on the operator-bundle-validate so seems like a good candidate for starters.

Reply:
Yes I need a help creating the image for that requirements, I should rephrase my sentence. We need container that fails running the playbook: validate-operator-bundle.yml.
The image quay.io/cvpops/test-operator:invalid-dependencies-v1 passes both playbooks.

Reply:
I currently don't have other option than setting wrong variable name specific only for the playbook. And that does not cover the case that is described in story.

Reply:
So after shord investigation the only way to fail the ansible playbook on purpose would be just providing incorrect variables before we run the playbook. So we already have it for setting up the non-existend folder for the work_dir. I think this test can be left as it is.

Reply:
Do we agree on leaving it the way we currently have? Specifying wrong env variable that is used only for the one of the playbooks "WORK_DIR" since there is no other way of simulating such situation.

Reply:
I think we can resolve this conversation based on the code changes that were made.